test/e2e/scenarios/valid-postcode-pages/invalidPostcode.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const paths = require('paths');
+const content = require('valid-postcode-pages/postcode-invalid/content.en.json');
+
+Feature('Invalid postcode');
+
+Before((I) => {
+    I.amOnPage(paths.validPostcode.invalidPostcode);
+});
+
+Scenario('When I go to the invalid postcode page I see the page heading', (I) => {
+
+    I.see(content.title);
+
+});
+
+Scenario('When I click the continue to form download link, I am taken to the form on external site', (I) => {
+
+    I.click(content.formDownload);
+    I.seeInCurrentUrl('https://hmctsformfinder.justice.gov.uk/HMCTS/GetForm.do?original_id=3038');
```

Comment:
I mentioned this to David yesterday, we need stop defining urls like this on the fly, they should be defined in a new urls.js file and reused throughout. We can let this through this time as we have SSCS-2370 in place.